code/modules/research/techweb/all_nodes.dm

```diff
@@ -549,9 +551,11 @@
 		"sheetifier",
 		"weldingmask",
 	)
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
-	required_experiments = list(/datum/experiment/ordnance/gaseous/bz)
-	discount_experiments = list(/datum/experiment/scanning/random/material/medium/one = 4000)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 15000)
+	discount_experiments = list(
+		/datum/experiment/scanning/random/material/medium/one = 4000,
+		/datum/experiment/ordnance/gaseous/bz = 10000,
+	) //ex-required experiments have large costs as a temp measure to see how often they get done when not a hard requirement.
```

Comment:
no historical comments. only describe what the code does now

Reply:
Righto, I'll take them out